assets/js/store/index.js

```diff
@@ -15,6 +16,9 @@ const storeOptions = {
     config: config,
     fetching: false
   },
+  modules: {
+    bearMessage
```

Comment:
Prefixing modules by project name it's not best approach. "systemMesseges" would fit better I think as it says more what are this messages.